test/CommentTester.cs

```diff
@@ -100,18 +100,22 @@ FOR INSERT
 ";
 
         private readonly string AutoGenerateComment = String.Format(
```

Comment:
If you want to go with that approach you could have an additional test that confirms that both strings are the same, that will catch divergence.